public/javascript/getLeagueInfo.js

```diff
@@ -0,0 +1,169 @@
+/**
+ * @file getLeagueInfo.js
+ * Handler for getting league match info.
+ *
+ */
+'use strict'
+ 
+var request = require('request');
+var MetaInspector = require('node-metainspector');
+var cheerio = require('cheerio');
+
+var userAPI, userThreadID, endTyping, returnString;
```

Comment:
Yeah I want them to be global scoped so other functions can access them without passing them around via parameters.